services/content-publishing/apps/worker/src/publisher/publishing.service.ts

```diff
@@ -112,6 +112,7 @@ export class PublishingService extends BaseConsumer implements OnApplicationBoot
     if (await this.publishQueue.isPaused()) {
       this.logger.debug('Received capacity.available event');
     }
+    await this.publishQueue.resume();
```

Comment:
Move it inside try ?

Reply:
I don't think it's necessary; the try is there to catch & ignore if the timeout is non-existent; other errors should bubble up.

Reply:
I believe `resume()` is a no-op if the queue is not paused